boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.dts

```diff
@@ -72,3 +72,11 @@
 &flexcomm4_lpuart4 {
 	status = "okay";
 };
+
+&flexspi {
+	status = "okay";
+};
+
+&w25q64jvssiq {
```

Comment:
This should not be needed- the flash chip will default to `status="okay"` unless it is explicitly set to disabled. We could also set the flash to disabled in the `frdm_mcxn947.dtsi`

Reply:
I have made this change to `frdm_mcxn947.dtsi` file.